connectors/sources/confluence.py

```diff
@@ -259,23 +259,31 @@ async def paginated_api_call(self, url_name, **url_kwargs):
         Yields:
             response: JSON response.
         """
-        url = os.path.join(self.host_url, URLS[url_name].format(**url_kwargs))
+        base_url = os.path.join(self.host_url, URLS[url_name].format(**url_kwargs))
+        start = 0
+
         while True:
             try:
+                url = f"{base_url}&start={start}"
```

Comment:
This line probably needs to be outside of the `while True`, otherwise it'll overwrite the last iteration's attempts to set the URL to the "next" link, if that strategy was used.